Tools/Scripts/webkitpy/__init__.py

```diff
@@ -64,7 +64,11 @@
 
 AutoInstall.register(Package('atomicwrites', Version(1, 1, 5)))
 AutoInstall.register(Package('attr', Version(20, 3, 0), pypi_name='attrs'))
-AutoInstall.register(Package('bs4', Version(4, 9, 3), pypi_name='beautifulsoup4'))
+
+if sys.version_info >= (3, 0):
+    AutoInstall.register(Package('bs4', Version(4, 10, 0), pypi_name='beautifulsoup4'))
```

Comment:
```suggestion
    AutoInstall.register(Package('bs4', Version(4, 12, 0), pypi_name='beautifulsoup4'))
```

we can't go beyond 4.12.0 because after that, from the 4.12.1 changelog:

> This version of Beautiful Soup replaces setup.py and setup.cfg with pyproject.toml. Beautiful Soup now uses tox as its test backend and hatch to do builds.

(c.f. https://bugs.webkit.org/show_bug.cgi?id=261082)

Reply:
Sure, I can try 4.12.

Reply:
Works for me.